MWC4K 2021 news post/wiki article (#5792)
ppy/osu-wiki: news/2021-07-07-osumania-4k-world-cup-2021-registrations-now-open.md

| Semifinals | Aug 28 & Aug 29 |
| Finals | Sep 4 & Sep 5 |
| Grand Finals | Sep 11 & Sep 12 |

## Prizes

- **1st:** $150 per team member, profile badge, "osu!mania Champion" user title for one year
- **2nd:** $80 per team member, profile badge
- **3rd:** $40 per team member, profile badge

## How do I register?

- **[Click here to sign-up!](https://osu.ppy.sh/community/tournaments/29)**
- Do not create teams in advance! Teams will be formed by each country's team captain after the Registration Phase is concluded.
- Submit your registration before **July 22.**

*Notice: you may only participate if your osu!mania 4K Global Ranking is #5000 or higher, and you have not violated the [osu! community rules](/wiki/Rules) in the past 12 months*.

If your registration passes our internal checks, you will be placed on the candidate list for your country. When that country's captain has been assigned, the list will be sent to them. This may take a while, so please be patient.

*Remember that any registrations submitted after the deadline will not be considered under any circumstances — submit your registration ahead of schedule to avoid any last-minute issues.*

## New leadership

> Each World Cup comes with new and exciting things, and this time around will be no exception. With our previous organisers juankristal and WalterToro stepping down it leaves some large shoes to fill. Starting from today, ChillierPear, LeoFLT, and Azer will be stepping up to meet the challenges and expectations as the new organisers of the osu! World Cups.

Reviewer: ```suggestion
Each World Cup comes with new and exciting things, and this time around will be no exception. With our previous organisers [juankristal](https://osu.ppy.sh/users/443656) and [WalterToro](https://osu.ppy.sh/users/5281416) stepping down, large shoes are to be filled. Starting from today, [ChillierPear](https://osu.ppy.sh/users/9501251), [LeoFLT](https://osu.ppy.sh/users/3668779), and [Azer](https://osu.ppy.sh/users/2155578) will be stepping up to meet the challenges and expectations as the new organisers of the osu! World Cups.
```

> "The three of us are very excited for this opportunity. In recent years, most, if not all of our time on osu! has been dedicated towards developing and nurturing the tournament scene. The osu! World Cups will still continue as they have been, and we hope to take them in a forward direction through the coming years and continue to provide the consistently high quality and entertaining experience that has existed for so long under the direction of our predecessors."

Reviewer: ```suggestion
> The three of us are very excited for this opportunity. In recent years, most, if not all of our time on osu! has been dedicated towards developing and nurturing the tournament scene. The osu! World Cups will still continue as they have been, and we hope to take them in a forward direction through the coming years and continue to provide the consistently high quality and entertaining experience that has existed for so long under the direction of our predecessors.
```
